server/data/services/githubService.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios');
+
+class GithubService {
+    static login() {
+        return `https://github.com/login/oauth/authorize?client_id=${process.env.GITHUB_CLIENT_ID}`;
+    }
+
+    static async authorize(code) {
+        const redirectURL = 'https://github.com/login/oauth/access_token';
+        const response = await axios.post(
+            redirectURL,
+            {
+                client_id: process.env.GITHUB_CLIENT_ID,
+                client_secret: process.env.GITHUB_CLIENT_SECRET,
+                code
+            },
+            {
+                headers: { Accept: 'application/json' }
+            }
+        );
+        return response.data.access_token;
+    }
+}
+
+module.exports = GithubService;
```

Comment:
As a general convention, we should name the files the same as the class/object/module that is exported (unless the component is a directory, then name it index.js or the NameOfComponentClass.js). I noticed it in other files in this PR, I think you should fix these all in a follow up PR.

Reply:
Will do this!